WordPress/src/main/res/layout/reader_cardview_post.xml

```diff
@@ -148,6 +148,8 @@
             android:paddingTop="@dimen/margin_extra_large"
             android:includeFontPadding="false"
             android:textAlignment="viewStart"
+            android:ellipsize="end"
+            android:maxLines="3"
```

Comment:
Suggestion (💡): Since both this `text_title` and the `reader_cardview_xpost.xml`'s such view are the only ones that are using the `ReaderTextView.Post.Title` style, both the `android:ellipsize="end"` and `android:maxLines="3"` can be moved to the style itself, thus forcing consistency and making those two views less complex.

PS: The only thing that needs to be done is adding the `<item name="android:ellipsize">end</item>` attribute to the `ReaderTextView.Post.Title` style. The `<item name="android:maxLines">3</item>` is already available there as such it is actually being duplicated when reused directly on the view level.
PSS: And as part of the above change, the `<item name="android:ellipsize">end</item>` attribute from the child `ReaderTextView.Post.Title.Detail` style can be then removed as it will be inherited by the parent `ReaderTextView.Post.Title` style. The child `<item name="android:maxLines">5</item>` overwrites the parent `<item name="android:maxLines">3</item>` and thus this is required, but the `<item name="android:ellipsize">end</item>` is not required and thus can/should be removed.

Reply:
Good suggestions: in deff17c if you have the time to double check :)

Reply:
👋 @zwarm !

Thanks for applying my suggestion! 👍

There is one last thing that you can do here, remove `android:ellipsize="end"` and `android:maxLines="3"` from the `reader_cardview_post.xml` layout as well.

Reply:
That would be helpful!! See I needed that double check. 🙇  @ParaskP7

Reply:
Great, it is looking good now, thanks so much @zwarm ! 🚀